Kernel/KerrGeoPlunge.m

```diff
@@ -0,0 +1,544 @@
+(* ::Package:: *)
+
+(* ::Title:: *)
+(*KerrGeoOrbit subpackage of KerrGeodesics*)
+
+
+(* ::Chapter:: *)
+(*Define usage for public functions*)
+
+
+BeginPackage["KerrGeodesics`KerrGeoPlunge`",
+	{"KerrGeodesics`ConstantsOfMotion`",
+	 "KerrGeodesics`OrbitalFrequencies`",
+	 "KerrGeodesics`SpecialOrbits`",
+	 "KerrGeodesics`FourVelocity`"}];
+
+
+KerrGeoPlunge::usage = "Takes either KerrGeoPlunge[a, Generic, En ,\[ScriptCapitalL],\[ScriptCapitalQ]] or KerrGeoPlunge[a, ISSO , RI] for generic Plunges or ISSO plunges respectively and returns a KerrGeoPlungeFunction[..] which stores the orbital trajectory and parameters. Here the ISSO plunges are paramaterised in terms of the choice of the radius of the ISSO for a given a there are range of allowed RI's which correspond to differeing inclinations in the prograde and retrograde directions, if an RI outside this range is given the used is provided with the range of allowed values to re-run the funciton.";
```

Comment:
We need to think carefully about this interface. Do we want separate functions for ISSO plunges? Or can everything be inferred automatically and unambiguously from context?

Reply:
I've reworded this section slightly; the usage is as follows. 

For generic plunges you provide either KerrGeoPlunges[a, {e,L,Q}] or KerrGeoPlunges[a, {e,L,Q}, {t0,to,theta0,phi0}] if you want to include initial conditions

For ISSO, one provides either KerrGeoPlunges[a, "ISSORadialParam", RI] or  KerrGeoPlunges[a, "ISSOIncParam", theta_Inc] with IC's working the same as the generic case as a final option.

The package defaults to Generic unless the user explicitly inputs an ISSO option which I think should be sufficiently unambiguous but I'm happy to consider alternatives.

Reply:
I agree the main issue will be deciding on the interface. This should probably go along with the discussion related to issue #37 and associated pull request #38. We were thinking of having default input parameters as (a,p,e,x) but then allowing associations to be passed if the user wishes to use alternative parameterizations.

We also need to decide if we want `KerrGeoPlunges` or to wrap this within `KerrGeoOrbit`